kge/util/io.py

```diff
@@ -22,7 +22,10 @@ def get_checkpoint_file(config: Config, checkpoint_argument: str = "default"):
         else:
             last_epoch = config.last_checkpoint_number()
```

Comment:
I think it would be more consistent if this would return 0 for the initial checkpoint (and the checkpoint were named _00000 as all other checkpoints). Whether or not the initial checkpoint should be retained no matter what may be a config option (default false?).

Reply:
renamed, but I still have to check the delete conditions, so it makes sense